api/seeders/20240724090342-add-city-to-recruiter_profiles.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  up: async (queryInterface) => {
+    const city = 'Odesa';
+    const email = 'mykyta.ganzha@gmail.com';
+
+    try {
+      await queryInterface.sequelize.query(`
+        UPDATE "recruiter_profiles"
+        SET city = :city
+        WHERE "user_id" IN (
+          SELECT id FROM "users" WHERE email = :email
+        )
+      `, {
+        replacements: { city, email },
+      });
+    } catch (err) {
+      throw new Error(`Error updating recruiter profiles: ${err.message}`);
+    }
+  },
+
+  down: async (queryInterface) => {
+    const email = 'mykyta.ganzha@gmail.com';
```

Comment:
`email` constant could be defined only once for `up` and `down` transactions
to make sure it is the same, it is easy to make a mistake in a sting literal and it is hard to find a bug then